deploy/kubernetes/overlays/noauth-debug/kustomization.yaml

```diff
@@ -8,6 +8,7 @@ transformers:
 patchesStrategicMerge:
 - noauth.yaml
 - controller-overlay.yaml
+- node-overlay.yaml
```

Comment:
why we need add this yaml too?

Reply:
this is a dev only file, I used it to deploy the GCE PD CSI driver with `AlwaysPull`